crates/fluvio/src/consumer/stream.rs

```diff
@@ -242,6 +242,7 @@ impl Drop for OffsetManagement {
             if let Err(err) = offset_store.try_flush() {
                 warn!("flush on drop failed: {err:?}");
             }
+            debug!("offsets flushed on drop");
```

Comment:
let's make info and add other information to figure out what's going on

Reply:
sure, I am printing the local offset store now